src/portable/st/synopsys/dcd_synopsys.c

```diff
@@ -570,16 +585,28 @@ bool dcd_edpt_open (uint8_t rhport, tusb_desc_endpoint_t const * desc_edpt)
     // "USB Data FIFOs" section in reference manual
     // Peripheral FIFO architecture
     //
+    // The FIFO is split up in a lower part where the RX FIFO is located and an upper part where the TX FIFOs start.
```

Comment:
Why do you include this comment twice?

Reply:
The FIFO layout was mentioned at this position before and i only updated it. It is a simply copy and paste issue. I will remove it!